build/runtimes.props

```diff
@@ -1,7 +1,7 @@
 <Project>
 
   <ItemGroup>
-    <!-- Workaround https://github.com/aspnet/AspNetCore/issues/6726. Required because illink (part of Microsoft.AspNetCore.Blazor.Mono) depends on .NET Core 2.x -->
+    <!-- BaselineGenerator is netcoreapp2.1. Other build tools may also require 2.x. -->
```

Comment:
I'm not positive we'll remember the logic behind installing this .NET runtime even with this comment. Suggest expanding the comment a bit.

Reply:
I'd be happy to change it, but don't really know what extra clarification would be useful. It already states what specific build item requires this runtime (`BaselineGenerator`). I'll proceed with it as-is, but if you want to expand the comment with particular phrasing please either do so, or let me know what change you want and I'll do it.